test/HealthChecks.MySql.Tests/Functional/MySqlHealthCheckTests.cs

```diff
@@ -1,11 +1,39 @@
 using System.Net;
+using MySqlConnector;
 
 namespace HealthChecks.MySql.Tests.Functional;
 
 public class mysql_healthcheck_should
 {
     [Fact]
-    public async Task be_healthy_when_mysql_server_is_available()
+    public async Task be_healthy_when_mysql_server_is_available_using_data_source()
+    {
+        var connectionString = "server=localhost;port=3306;database=information_schema;uid=root;password=Password12!";
+
+        var webHostBuilder = new WebHostBuilder()
+            .ConfigureServices(services =>
+            {
+                services
+                    .AddMySqlDataSource(connectionString)

```

Comment:
Is there any chance you could add a `README.md` file similar to https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/master/src/HealthChecks.Azure.Data.Tables/README.md that recommends using this health check with the registration of data source like this?